components/core/src/clp_s/CommandLineArguments.hpp

```diff
@@ -21,7 +21,12 @@ class CommandLineArguments {
     };
 
     // Constructors
-    explicit CommandLineArguments(std::string const& program_name) : m_program_name(program_name) {}
+    explicit CommandLineArguments(std::string const& program_name)
+            : m_program_name(program_name),
+              m_compression_level(3),
+              m_target_encoded_size(8UL * 1024 * 1024 * 1024),  // 8 GB
+              m_mongodb_enabled(false),
+              m_batch_size(1000) {}
```

Comment:
* Our guideline is to set defaults for members at the declaration site (i.e., the member declaration in the header file).
* This is very minor, but to be future proof, we should write `8 GiB`

Reply:
For the first point I think clp does the same thing (set values in the constructor)? Do we also want to change it for clp later?

Reply:
Yep.